### PR TITLE
feat: document vm.toString()

### LINKED ISSUE #407: Document `vm.toString`
See https://github.com/foundry-rs/foundry/pull/2127

### PR DESCRIPTION
**Pull request by odyslam**
fixes #407

### CODE REVIEW

File: src/cheatcodes/to-string.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+function toString(int256) external returns (string memory);
+function toString(bytes32) external returns (string memory);
+function toString(bytes) external returns (string memory);
+```
+
+### Description
+
+Convert any type to it's string version. Very useful for operations that demand strings, such as the cheatcode `ffi`.
+
+Bytes are converted to a string of their hex representation.
```
Comment (onbjerg) [inline src/cheatcodes/to-string.md:18]: With or without 0x?

**Review by onbjerg — APPROVED** [review]
LGTM when last comment is addressed :smile:

File: src/cheatcodes/to-string.md (hunk 1 above)
Comment (odyslam) [reply src/cheatcodes/to-string.md:18]: With 0x. Let me update the docs a. bit.

**Comment by odyslam** [comment]
@onbjerg addressed. I think I can't merge?

File: src/cheatcodes/to-string.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+## `toString`
+
+### Signature
+
+```solidity
+function toString(address) external returns (string memory);
```
Comment (mds1) [inline src/cheatcodes/to-string.md:6]: Is the returned address a checksum address (IMO it should be 🤞)

Comment (odyslam) [reply src/cheatcodes/to-string.md:6]: At the moment it isn't. That's a good observation.

Comment (odyslam) [reply src/cheatcodes/to-string.md:6]: Fixed

**Pull request merged**